.github/scripts/commons.mjs

```diff
@@ -8,17 +8,20 @@ export const config = {
   testAssetsDir: `./test-assets/`,
 }
 
+export const branchName = process.env.BRANCH_NAME || 'main';
+
 export async function cloneWcagActRules({ tmpDir }) {
   await $`rm -rf ${tmpDir}`;
   await $`git clone \
     --no-single-branch \
     --branch main \
-    https://github.com/w3c/wcag-act-rules.git ${tmpDir} \
+    https://github.com/daniel-montalvo/wcag-act-rules.git ${tmpDir} \
```

Comment:
It looks a bit weird that this will setup something in the `daniel-motvaldo` organisation 🤔 Shouldn't it stay in the `w3c` one?
(not sure whether this will only be run for the branch deploy or also the regular ones)